WooCommerce/Classes/ViewRelated/CardPresentPayments/BuiltInCardReaderConnectionController.swift

```diff
@@ -0,0 +1,769 @@
+import Combine
```

Comment:
This class looks massive, probably because of the enums and comments inside, but I think it's doing many things in one place. Did you consider splitting it into several other classes/files to make it easier to maintain and work with?